tests/dummy/app/mixins/edit-form-mixin.js

```diff
@@ -1,9 +1,11 @@
-import EditFormRoute from 'ember-flexberry/routes/edit-form';
+import Ember from 'ember';
 
-/**
-  Base edit form route for dummy application.
- **/
-export default EditFormRoute.extend({
+export default Ember.Mixin.create({
+  /**
+  * Сleaning of the changes on edit form.
```

Comment:
By the team agreement we comment like: 
/**
There we do not use star marks.
*/